helm/reana/templates/reana-db.yaml

```diff
@@ -32,6 +32,9 @@ spec:
       containers:
       - name: db
         image: postgres:9.6.2
+        args:
+        - -c
+        - max_connections=300
```

Comment:
How to test the PR:

1. Edit the value to your liking, e.g. I was using 1000 for local stress tests.
2. Redeploy REANA cluster.
3. Check the value via:

```console
$ kubectl exec -i -t deployment/reana-db -- psql -U reana -c 'SHOW max_connections'
 max_connections 
-----------------
 1000
```